src/main/java/nl/jiankai/mapper/converters/StructToObjectArrayConverter.java

```diff
@@ -0,0 +1,32 @@
+package nl.jiankai.mapper.converters;
+
+import nl.jiankai.annotations.Converter;
+
+import java.sql.SQLException;
+import java.sql.Struct;
+
+@Converter(autoApply = true)
+public class StructToObjectArrayConverter implements AttributeConverter<Struct, Object[]> {
+
+    @Override
+    public Object[] convert(Struct value) {
+        Object[] convertedValue = new Object[0];
+    try {
+        convertedValue = value.getAttributes();
```

Comment:
This is not really useful since you're just returning the attributes which is an array of objects. The instance field in the domain models its mapping to will almost always be another object (namely the java model representation of the underlying sql model it's referencing).